P5/Source/Guidelines/en/ST-Infrastructure.xml

```diff
@@ -1370,6 +1371,7 @@ entities in the XML DTD fragments.--></p>
         values). In this latter case, the datatype will not be <ident type="datatype">teidata.enumerated</ident>, but an explicit list of the possible values.</p>
       <specGrp xmlns:rng="http://relaxng.org/ns/structure/1.0" xml:id="DTYPES-4" n="Coded value datatypes">
         <xi:include href="../../Specs/teidata.word.xml"/>
+        <xi:include href="../../Specs/teidata.gender.xml"/>
```

Comment:
**THIS IS AN ERROR**
The definitions of teidata.gender and teidata.sex are now teidata.enumerated; therefore they need to be included _after_ teidata.enumerated is. (Lest DTDs break.)

Reply:
Should be fixed in https://github.com/TEIC/TEI/commit/3dc6914cf5b762e1c54f2c52b31609d074d85aee